src/funsnake/App.fs

```diff
@@ -182,12 +202,10 @@ let main() =
             moveDone <- false
         :> obj)
 
-    // Capture MSPointerDown event for IE to move the snake (touch events for Chrome & iOS are in the html code)
-    canvas.addEventListener("pointerdown", unbox(fun e ->
-        let ev= unbox<MSPointerEvent> e
-        touch <- (ev.pageX - canvas.offsetLeft, ev.pageY - canvas.offsetTop)))
-
-    // Draw the walls only once
+    // Capture touch events
+    setTouchListener
```

Comment:
`setTouchListener` is a value, so it's eagerly evaluated and this doesn't do anything. You could convert it into a function call `setTouchListener()` (also changing the declaration to `let setTouchListener () =`)